lib/dslash_policy.cuh

```diff
@@ -1946,7 +2060,7 @@ public:
      popKernelPackT();
    }
 
-   int tuningIter() const { return 10; }
+   int tuningIter() const { return 20; }
```

Comment:
Can this be reduced back down to 10? I'm thinking about how long the nvshmem policies take to tune as it is. If this is empirically worth it for stability I'm happy to keep it this way.

Reply:
This is only for the final policy tuning. What takes time is the exploded tune space for the fused kernels.